### ISSUE TITLE
Double Data Rate (DDR) type wrong

### ISSUE THREAD

**Issue by Revadike** [Carlmundo/SysID#1]
It says I have DDR3, but I have DDR4.

**Comment by Carlmundo** [comment]
Thanks, I'll fix this soon.

**Comment by Revadike** [comment]
> soon

Carlmundo commented 22 days ago 😏

**Comment by Carlmundo** [comment]
Fixed in 74843056a4f08077a29668724735b710e2acaeb3

**Comment by Revadike** [comment]
Thanks! 
```
Operating System: Windows 10 Education
Processor: AMD Ryzen 7 2700X Eight-Core Processor   
RAM: 32 GB (Max. 256 GB)

Total Slots: 4
Free Slots: 2

Slot 1: 16 GB DDR4 3200Mhz
Slot 2: 16 GB DDR4 3200Mhz

Optical Drive: Yes
Graphics Card: NVIDIA GeForce GTX 1070
Microsoft Office: 2016
Free Space: 58.44 GB
Total Space: 231.82 GB

```
